zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -16,7 +16,7 @@ import zio.{Has, ZIO, ZManaged}
 abstract class HttpRunnableSpec(port: Int) extends DefaultRunnableSpec { self =>
```

Comment:
```suggestion
abstract class HttpRunnableSpec extends DefaultRunnableSpec { self =>
```

Reply:
We don't need to pass the port anymore for tests. Which would be great for tests.